tests/quickstart_test.py

```diff
@@ -0,0 +1,87 @@
+import os
+import time
+import pytest
+from testbook import testbook
+import json
+from pytest_dependency import depends  # import the depends function
+from common_test import gcp_setup
+
+
+@pytest.fixture(scope="module")
+def tb():
```

Comment:
maybe rename to notebook()?

Reply:
I was adopting the `testbook` docs convention but sure nb makes good sense too.

Reply:
feel free to keep as-is

Reply:
yeah I started to go to nb and then saw again taht the fixture returns a `TestbookNotebookClient` instance.  I think `tb` makes good sense once you know that.